### DISCUSSION TITLE
Get all ressources

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#7687] [Help]
Hi Can i get all ressources from  a panel ? I want ton separate admin panel and app panel for example and i want ton administrate ressources app panel from admin panel

**Comment by Z3d0X** [comment]
```php
use Filament\Facades\Filament;

Filament::getPanel()->getResources();
```

**Comment by warsemann** [comment]
Thanks but in admin panel getPanel() give me admin panel not app ?

**Reply by Z3d0X** [reply]
```php
Filament::getPanel('app')
```
